### PR TITLE
redesign wp5 (profile page)

### PR DESCRIPTION
**Pull request by rewodev**
Applied design from [template](https://xd.adobe.com/view/a9a3821d-ff0a-4571-bf32-930048dda25d-fcf0/screen/4c8cc1c2-c422-4a79-a068-3711f96f5b11/specs/) to profile page. The landing page after login now routes to this page also.

Mobile layout was not specified (not found any design template) - changes welcome 😃

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -72,11 +70,11 @@ const routes: Routes = [
         path: 'archive',
         component: ArchiveComponent
       },
-      {
-        path: 'my-profile',
-        component: MyProfileComponent,
-        canActivate: [AuthGuard]
-      },
+      // {
+      //   path: 'my-profile',
```
Comment (SandroGlueck) [inline src/app/app-routing.module.ts:74]: You removed the route 'my-profile' but you still navigate to it in various different places - is that intended?

**Review by SandroGlueck — COMMENTED** [review]
That was quick 🚀 
Thanks for the great work.

Don't really have anything to complain about. Only the Routing (/my-profile) seems odd.

I will also look into the merge conflicts - so no need to worry about those.

**Comment by SandroGlueck** [comment]
image

The letter in the "profile picture" on the my-profile component is lowercase and therefore not centered.
Might be worthwhile to create a component out of that, as it is used in the header and the profile now.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (rewodev) [reply src/app/app-routing.module.ts:74]: My bad - I will fix that.

**Review by PotSchmoell — CHANGES_REQUESTED** [review]
Looking great functionality wise. I only have a few little comments/questions:
- Can you include an anchor at the case slider on the profile page and link to that anchor from the "Fall bearbeiten" in the header
- Have you checked, whether the layout of the "Deine abgeschlossenen Fälle" and "In gelösten Fällen stöbern" sections works, when there are more elements?
- Considering the previous point: Can we make the hero-illustration a little smaller? Or would that be too small when there are more elements in the list on the right? 
- On mobile, there seems to be something wrong with the layout - Maybe it's because the headline (username) is too large?
image

image

**Comment by rewodev** [comment]
@PotSchmoell 

- layout is working for multiple items
- illustration reduced to 280px size looks so:

image

Concerning the anchor: If user clicks on "Fall bearbeiten", view should go to profile page and show case slider. Right?

**Comment by PotSchmoell** [comment]
> Concerning the anchor: If user clicks on "Fall bearbeiten", view should go to profile page and show case slider. Right?

Exactly.

**Review by PotSchmoell — APPROVED** [review]

**Comment by SandroGlueck** [comment]
> FAIL src/app/my-profile/components/my-profile/my-profile.component.spec.ts (69.476 s)

This file causes the tests to fail

**Pull request merged**